pyproject.toml

```diff
@@ -117,7 +117,6 @@ pyparsing = "^3.1.1"
 flask-wiki = "^0.3.1"
 dnspython = ">2.4.2"
 cryptography = ">41.0.7"
-babel = "<2.15.0"  # NOTE: remove when https://github.com/python-babel/babel/issues/1081 is solved
```

Comment:
babel < 2.15.0 will broke the test, thus we need to add a constraint to force babel > 2.14.0

Reply:
This PR will correct the babel tests.

Reply:
Not for babel 2.14.0 and lower, that is why to be completely correct with the tests, we should add `babel = ">2.14.0"`